mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_service_test.go

```diff
@@ -0,0 +1,100 @@
+package apphub_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestDataSourceApphubDiscoveredService_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
```

Comment:
Should we append random_suffix to resources created in the tests - to allow for parallel test runs?

Reply:
done